scripts/docs_version_pragma_check.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+
+# ------------------------------------------------------------------------------
+# This file is part of solidity.
+#
+# solidity is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# solidity is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with solidity.  If not, see <http://www.gnu.org/licenses/>
+#
+# (c) 2016 solidity contributors.
+#------------------------------------------------------------------------------
+
+set -e
+
+## GLOBAL VARIABLES
+
+REPO_ROOT=$(cd $(dirname "$0")/.. && pwd)
+SOLIDITY_BUILD_DIR=${SOLIDITY_BUILD_DIR:-build}
+source "${REPO_ROOT}/scripts/common.sh"
+source "${REPO_ROOT}/scripts/common_cmdline.sh"
+
```

Comment:
Please document: This script verifies that the examples compile with the oldest version mentioned in the pragma. It does not verify that it cannot be compiled with an older version and it also does not verify that it can be compiled with the newest version compatible with the pragma.